microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,148 @@
+
+
+<!-- ====================================================================== -->
+## Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View
+
+<!-- Title: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View -->
+<!-- Subtitle: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. -->
+
+Several improvements to [Focus Mode](/devtools-guide-chromium/experimental-features/focus-mode) in existing location controls for DevTools have been made, including a new control for changing the orientation of **Quick View**.
+
+The Settings menu now provides options for changing DevTools between docked locations and a separate window. This menu also contains settings for how the **Activity Bar** is displayed. Starting in Edge 105, these options can be navigated and changed more quickly. The previous submenus have been replaced with toggle buttons. Additionally, improved contrast means the option you have selected is more easily visible at a glance.
+
+Before:
+
+![Focus Mode Before](./devtools-105-images/before-docking-menu.png) 
+
+After:
+
+![Focus Mode After](./devtools-105-images/after-docking-menu.png)
+
+You can now change the orientation of the **Quick View** panel as well. To change Quick View to a vertical split, select the location toggle (**Dock Quick View to the right**). Selecting this toggle a second time will return **Quick View** to a horizontal split.
+
+![Quick View from Console](./devtools-105-images/quickview-console.png) 
+
+To minimize **Quick View**, use the **Collapse** toggle or press `Esc`:
+
+![Focus Mode Improved Location Control](./devtools-105-images/focus-mode-improved-location-controls.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+<!-- ====================================================================== -->
+
+## Fix: Search in the Sources and Network tools now works as expected in Focus Mode
+
+<!-- Rank 4 -->
+<!-- Title: Fix: Search in the Sources and Network tools now works as expected in Focus Mode  -->
+<!-- Subtitle: Try Focus Mode, a new, more simplified, and streamlined UI for DevTools!-->
+
+![Focus Mode experiment](/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode#enable-focus-mode) in previous versions of Microsoft Edge didn't properly showcase the Search feature in the Network tool nor the *Search in all files* feature in the Sources tool. In Microsoft Edge version 105, these issues have been fixed.
+
+You can now search in the **Network** tool via the **Search** icon.  
+
+![Networking Search](./devtools-105-images/networking-search.png)
+
+You can leverage the *Search in all files* feature in the Sources tool by opening the context menu on files in the Page section.  This will display the Search tool in the **Quick View**.
```

Comment:
Same comment as earlier. Why italicize context menu entries? I think we should be consistent

Reply:
fixed in recent commit, thus Outdated comment